<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GGStyleDialog</class>
 <widget class="QDialog" name="GGStyleDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>467</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Styles</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Basic style</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Font</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QFontComboBox" name="cmbFont"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Size (pt)</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="spnPointSize"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Background</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="lblBack">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::Box</enum>
          </property>
          <property name="text">
           <string notr="true"/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="btnChangeBack">
          <property name="text">
           <string notr="true"/>
          </property>
          <property name="icon">
           <iconset>
            <normalon>:/icons/edit</normalon>
           </iconset>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Foreground</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="lblFore">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::Box</enum>
          </property>
          <property name="text">
           <string notr="true"/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="btnChangeFore">
          <property name="text">
           <string notr="true"/>
          </property>
          <property name="icon">
           <iconset>
            <normalon>:/icons/edit</normalon>
           </iconset>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Styles</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="0,1">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QListWidget" name="lstStyles">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QToolButton" name="btnRemoveStyle">
            <property name="text">
             <string notr="true"/>
            </property>
            <property name="icon">
             <iconset>
              <normalon>:/icons/delete</normalon>
             </iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="btnAddStyle">
            <property name="text">
             <string notr="true"/>
            </property>
            <property name="icon">
             <iconset>
              <normalon>:/icons/add</normalon>
             </iconset>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QFormLayout" name="formLayout_2">
        <item row="1" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Foreground</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="lblStyleFore">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
            <property name="frameShape">
             <enum>QFrame::Box</enum>
            </property>
            <property name="text">
             <string notr="true"/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="btnChangeStyleFore">
            <property name="text">
             <string notr="true"/>
            </property>
            <property name="icon">
             <iconset>
              <normalon>:/icons/edit</normalon>
             </iconset>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Name</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="txtStyleName"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>GGStyleDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>257</x>
     <y>457</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>GGStyleDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>457</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lstStyles</sender>
   <signal>currentRowChanged(int)</signal>
   <receiver>GGStyleDialog</receiver>
   <slot>showStyle()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>63</x>
     <y>291</y>
    </hint>
    <hint type="destinationlabel">
     <x>0</x>
     <y>292</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnAddStyle</sender>
   <signal>clicked()</signal>
   <receiver>GGStyleDialog</receiver>
   <slot>addStyle()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>262</x>
     <y>406</y>
    </hint>
    <hint type="destinationlabel">
     <x>2</x>
     <y>416</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cmbFont</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>GGStyleDialog</receiver>
   <slot>basicStyleChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>334</x>
     <y>42</y>
    </hint>
    <hint type="destinationlabel">
     <x>472</x>
     <y>41</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spnPointSize</sender>
   <signal>editingFinished()</signal>
   <receiver>GGStyleDialog</receiver>
   <slot>basicStyleChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>407</x>
     <y>61</y>
    </hint>
    <hint type="destinationlabel">
     <x>477</x>
     <y>78</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>txtStyleName</sender>
   <signal>editingFinished()</signal>
   <receiver>GGStyleDialog</receiver>
   <slot>currentSyleRenamed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>430</x>
     <y>188</y>
    </hint>
    <hint type="destinationlabel">
     <x>477</x>
     <y>187</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnRemoveStyle</sender>
   <signal>clicked()</signal>
   <receiver>GGStyleDialog</receiver>
   <slot>removeStyle()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>404</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>463</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>showStyle()</slot>
  <slot>removeStyle()</slot>
  <slot>addStyle()</slot>
  <slot>basicStyleChanged()</slot>
  <slot>currentSyleRenamed()</slot>
 </slots>
</ui>
